SbtProject/project/sbt-updates.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0")
```

Comment:
I understand that. My experience is that _doing that_ leads to unrepeatable builds, since everybody has their own plugins.

I wanna be able to pull code and compile, package, etc. stuff with it without having to manually get a plugin, 'cause that's not what I'm ever thinking about.